### PR TITLE
doc: Mention PR Club in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by cvengler**
As we have a ["Review Club"](https://github.com/bitcoin/bitcoin/labels/Review%20club) label it would be worth mentioning it in the CONTRIBUTING.md file for beginners.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ release cycle, overall merging, moderation and appointment of maintainers.
 
 If you're looking for somewhere to start contributing, check out the
 [good first issue](https://github.com/bitcoin/bitcoin/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
-list.
+list or participate at a weekly [Bitcoin Core PR Review Club](https://bitcoincore.reviews/) meeting.
```
Comment (michaelfolkson) [inline CONTRIBUTING.md]: nit: s/at/in

**Review by michaelfolkson — COMMENTED** [review]
ACK

File: CONTRIBUTING.md (hunk 1 above)
Comment (emilengler) [reply CONTRIBUTING.md]: ACK

Comment (emilengler) [reply CONTRIBUTING.md]: Fixed 34d826ea5f900b6320f6ff070c3632155acb716e

**Comment by laanwj** [comment]
ACK

**Comment by practicalswift** [comment]
ACK 34d826ea5f900b6320f6ff070c3632155acb716e -- Bitcoin Review Club is a great effort to introduce new contributors to the project in a positive and friendly way! We need that kind of newcomer friendly on-ramps to attract and educate the next generation of contributors! Kudos to @jnewbery for helping secure the project also in the super long-term :)

**Pull request merged**